app/Auth/Access/SocialAuthService.php

```diff
@@ -40,6 +40,9 @@ public function __construct(UserRepo $userRepo, Socialite $socialite, SocialAcco
     public function startLogIn($socialDriver)
     {
         $driver = $this->validateDriver($socialDriver);
+        if ($socialDriver == 'google' && env('GOOGLE_SELECT_ACCOUNT')) {
```

Comment:
The `env` function should not be directly used. Ideally a new `config/services.php` entry should be added and then that should be used instead (`config('services.google.select_account')`).

Also, Since the logic here is the same for both login and register it would be ideal to have this code extracted to it's own function within this service class.